lib/thumbs/pull_request_worker.rb

```diff
@@ -176,7 +176,9 @@ def reviews
 
     def debug_message(message)
       log = Log4r::Logger['Thumbs']
-      log.debug("#{@repo} #{@pr.number} #{@pr.state} #{message}")
+      if log
```

Comment:
I'll submit a PR later for reducing the de-ref'ing of the logger and subsequent conditional. In short, good catch, here to not log if there's no logger.